src/app/static/package.json

```diff
@@ -82,35 +82,34 @@
     "html-webpack-plugin": "^5.5.0",
     "i18next": "^19.0.1",
     "ify-loader": "^1.1.0",
-    "jest": "^27.5.1",
     "jest-canvas-mock": "^2.1.2",
     "jest-environment-jsdom": "^27.5.1",
     "jest-teamcity-reporter": "^0.9.0",
+    "jest": "^27.0.5",
+    "sass": "^1.32.7",
+    "sass-loader": "^12.0.0",
+    "ts-jest": "^27.0.4",
+    "typescript": "~4.5.5",
```

Comment:
Sorry, wasn't being clear.,

1. Why have some dependency versions been regressed? How is this related to the change to use vue-cli?
2. Just a general enquiry about why ts-jest has been introduced, since you didn't remark on this in the description. Is that also related to the change to vue-cli, or just a separate useful change?

Reply:
Looks like a couple of dependencies are still targeting lower versions than before:
"jest": "^27.0.5" (was "^27.5.1")
"sass-loader": "^12.0.0" (was "^13.2.2")
"ts-jest": "^27.0.4" (was "^27.1.4")
(sorry, that was already there, just didn't spot it as dep ordering has changed...)

Since these all use ^ this will only make a practical difference to `sass-loader` when updating the lock file, since that's the only one with a different major version.

Reply:
I think this is absolutely trivia but I have made changes.